deployment/aks-periscope-kind.yaml

```diff
@@ -0,0 +1,191 @@
+# Note: this file is deprecated and will be removed in a future release
```

Comment:
@Tatsinnit @arnaud-tincelin is the presence of this file the only outstanding concern with this PR? The other concerns raised on the previous version (https://github.com/Azure/aks-periscope/pull/69) were WRT logging format and usage of RunCommandOnHost - which are issues that I consider should be changed in separate PR's and consistently all-at-once rather than piecemeal. 

IMO this is fine to commit as is and we can refactor to a Kustomize version later. The reason I said "temp for development" on the previous PR was not intending that this be changed before merging, but that it would be eventually replaced with a Kustomize based version - just as the current aks-periscope.yaml will.